examples/Console/TestOtlpExporter.cs

```diff
@@ -53,12 +54,15 @@ internal static object Run(string endpoint)
 
         private static object RunWithActivitySource(string endpoint)
         {
+            // Enable support for unencrypted HTTP2
+            AppContext.SetSwitch("System.Net.Http.SocketsHttpHandler.Http2UnencryptedSupport", true);
+
             // Enable OpenTelemetry for the sources "Samples.SampleServer" and "Samples.SampleClient"
             // and use OTLP exporter.
             using var openTelemetry = Sdk.CreateTracerProviderBuilder()
                     .AddSource("Samples.SampleClient", "Samples.SampleServer")
                     .SetResourceBuilder(ResourceBuilder.CreateDefault().AddService("otlp-test"))
-                    .AddOtlpExporter(opt => opt.Endpoint = endpoint)
+                    .AddOtlpExporter(opt => opt.Endpoint = new Uri(endpoint))
```

Comment:
For the `netstandard2.1` build, I changed the Endpoint to be a `Uri` instead of `string`. This is because if the scheme is not present then the client throws:

```
System.ArgumentException: Only 'http' and 'https' schemes are allowed.
```

I figure using `Uri` helps mitigate this.